idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -351,4 +351,14 @@ service WorkflowService {
       3: shared.EntityNotExistsError entityNotExistError,
     )
 
+  /**
+  * GetPollerHistory returns pollers which poll from given tasklist in last few minutes.
+  **/
+  shared.GetPollerHistoryResponse GetPollerHistory(1: shared.GetPollerHistoryRequest request)
```

Comment:
I thought we want expose something like DescTaskList. For now it only include the poller info, but soon we are going to expose the task list queue size and task list wait time.

Reply:
i did just give some random name to the API and parameters, let us wait until @molteanu come back